backend/src/models/projects/client_coding.js

```diff
@@ -20,7 +20,7 @@ const findById = (id) => {
   return knex
     .select(
       "cc.*",
-      { client_amount: knex.raw("cc.client_amount::numeric::float8") },
+      { client_amount: knex.raw("cc.client_amount") },
```

Comment:
nice. i always thought those casts were smelly

Reply:
Lol they were necessary before we added the money type fields.